.github/workflows/main.yml

```diff
@@ -59,7 +61,13 @@ jobs:
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v4
-      - name: Run PA loadtest
+        with:
+          lfs: true
+      # TODO(timothytrippel): It is unclear why the `lfs: true` configuration
+      # above is not sufficient on self hosted runners.
+      - name: Initialize LFS objects
+        run: git lfs pull
+      - name: Run integration tests
```

Comment:
We probably need to come up with a lock file setup for the FPGA.

Reply:
Yes I can look into that in a follow-up.